src/Entity/Post.php

```diff
@@ -198,7 +198,6 @@ public function addComment(?Comment $comment): void
 
     public function removeComment(Comment $comment): void
     {
-        $comment->setPost(null);
```

Comment:
@yceruto, Why did you remove this line?

Reply:
See https://github.com/symfony/demo/pull/899#discussion_r235046319

Reply:
@yceruto, Ok, I got it.